Code/Tools/FBuild/FBuildCore/Graph/ObjectNode.cpp

```diff
@@ -1670,6 +1692,14 @@ void ObjectNode::GetExtraCacheFilePaths( const Job * job, Array< AString > & out
         outFileNames.Append( xmlFileName );
     }
 
+    // JSON files produced by -ftime-trace option
+    if ( objectNode->m_CompilerFlags.IsUsingClangTimeTrace() )
+    {
+        AStackString<> jsonFileName;
+        objectNode->GetTimeTraceJsonName( jsonFileName );
+        outFileNames.Append( jsonFileName );
```

Comment:
This will require the cache version to be bumped, otherwise older workers will break when retrieving older cache entries that don't have a 5th file in the result they retrieve.

It's ok to simply increment the cache version to fix this.